plugins/provisioners/ansible/provisioner/base.rb

```diff
@@ -134,6 +141,12 @@ def generate_inventory_groups
               defined_groups << gname
               inventory_groups += "\n[#{gname}]\n"
               gmembers.each do |gm|
+              # TODO : Expand and validate host range patterns
+              # against @inventory_machines list before adding them
+              # otherwise abort with an error message
```

Comment:
please fix comment indentation. We'll probably remove the comment and create a github issue, but it's fine for now.